custom_components/hass_govee_h6199/manifest.json

```diff
@@ -9,6 +9,6 @@
     "iot_class": "local_polling",
     "issue_tracker": "https://github.com/necrokote/hass-govee-h6199/issues",
     "loggers": ["custom_components.hass-govee-h6199"],
-    "requirements": ["bleak-retry-connector", "govee-h6199-ble"],
-    "version": "0.3.0"
+    "requirements": ["bleak-retry-connector", "govee-h6199-ble @ git+https://github.com/NecroKote/pygovee-h6199-ble@a6f2354"],
+    "version": "0.3.5"
```

Comment:
why +0.0.5 and not +0.1.0 ? 
this looks like significant enough change. 

But also, I'd rather have version bumps as separate commits. No biggie tho